frontend/components/wrapper/pages/CourseList.tsx

```diff
@@ -4,11 +4,12 @@ import React, { useState } from 'react'
 import { CourseList } from '../../pages/CourseList'
 import { CourseListType } from '../../../types/CourseType'
 import Error from '../../../app/error'
+import { AdvertisementType } from '../../../types/AdvertisementType'
 
 export function CourseListWrapper({
-  initialCourses,
+  initialCourses, advertisements
 }: {
-  initialCourses: CourseListType[]
+  initialCourses: CourseListType[], advertisements: AdvertisementType[]
```

Comment:
```suggestion
CourseListWrapperProps
```

みたいに定義して型を作りたいですね。

上のファイルの
```
type AdvertisementTypeProps = {
  advertisements: AdvertisementType[] | null | undefined
}
```
あたり参考にお願いします。